pkg/github/github.go

```diff
@@ -40,10 +40,34 @@ type Config struct {
 	maxRetryDelay     time.Duration
 }
 
+// Issue is the GitHub Issue.
+type Issue struct {
+	Number *int
+}
+
+type IssueComment struct {
+	ID *int64
+}
+
+var (
+	Closed = "closed"
+	Open   = "open"
+	Any    = "any"
```

Comment:
Doc. Also recommend `const` instead of `var`.

Reply:
I used var so that I could reference the ptr since we need a ptr in the api.

Reply:
Made them constants. Found a way. We use these outside of this module so I wanted to keep them public.